src/pages/index.jsx

```diff
@@ -1,95 +1,138 @@
+import Box from '@mui/material/Box'
+import Button from '@mui/material/Button'
 import Card from '@mui/material/Card'
-import CardActions from '@mui/material/CardActions'
 import CardContent from '@mui/material/CardContent'
 import CardMedia from '@mui/material/CardMedia'
+import Grid from '@mui/material/Grid';
 import Typography from '@mui/material/Typography'
 import Head from 'next/head'
 import Image from 'next/image'
 
-import orcas from '../../public/images/orca-breach.jpg'
+import arrow from '../../public/images/Arrow.png'
+import orcas from '../../public/images/homepage.png'
+import vector from '../../public/images/vector.png'
```

Comment:
@alisonqiu This is the reason why PR isn't passing all tests, you have added an image with name `Vector.png`, but you are using the name `vector.png` while importing, which is not there in images folder. So, you can change the import to `../../public/images/Vector.png`.

Reply:
@alisonqiu Also it would be better to have all the images to be kept inside a separate folder in the images folder.

Reply:
> @alisonqiu Also it would be better to have all the images to be kept inside a separate folder in the images folder.

Do you mean create a separate folder inside the images folder just to track the changes?